templates/default/apache_ckan.conf.erb

```diff
@@ -1,12 +1,9 @@
 <VirtualHost 127.0.0.1:8000>
     ServerName <%= @app_url %>
     ServerAlias www.<%= @app_url %>
-
-    # Use canonical domain
-    RewriteEngine on
```

Comment:
We still need `RewriteEngine on` in order to redirect non-English pages.

Reply:
done